tests/lib/collectors/azure_logging.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests the Azure logging collector."""
+
+import unittest
+import unittest.mock as mock
+from datetime import datetime as dt
+
+from dftimewolf.lib import state
+from dftimewolf import config
+from dftimewolf.lib.collectors import azure_logging
+from dftimewolf.lib.containers.containers import AzureLogs
+
+
+class AzureLogging(unittest.TestCase):
+  """Tests for the Azure logging collector."""
+
+  def testInitialization(self):
+    """Tests that the collector can be initialized."""
+    test_state = state.DFTimewolfState(config.Config)
+    azure_logging_collector = azure_logging.AzureLogsCollector(test_state)
+    self.assertIsNotNone(azure_logging_collector)
+
```

Comment:
I know it's tedious, but we should still test `SetUp` in here to make sure attributes are set correctly.